components/cellery-component-parent.ipr

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please remove this file from the commit. This is an Intellij IDEA project file.

Reply:
Removed